src/pagesNew/apps/Workspace/WorkspaceBody.tsx

```diff
@@ -0,0 +1,247 @@
+import {
+	Box,
+	Button,
+	ClickAwayListener,
+	IconButton,
+	InputAdornment,
+	InputLabel,
+	MenuItem,
+	Select,
+	TextField,
+	Tooltip,
+	Typography,
+} from "@mui/material"
+import { useState } from "react"
+import { LuSearch } from "react-icons/lu"
+import HelpOutlineIcon from "@mui/icons-material/HelpOutline"
+import { relative } from "path"
+import { ActiveBacground, BoardCreateTooltip } from "./BoardCreateTooltip"
+import { Link } from "react-router-dom"
+export type BoardType = {
+	title: string
+	backgroundImage: string
+	visibility: string
+}
+export const WorkspaceBody = () => {
+	const [open, setOpen] = useState(false)
+
+	const handleTooltipClose = () => {
+		setOpen(false)
+	}
+
+	const handleTooltipOpen = () => {
+		setOpen(true)
+	}
+	const [boardList, setBoardList] = useState<BoardType[]>([])
+
+	return (
+		<Box sx={{ m: 5 }}>
+			<Typography sx={{ fontSize: "1.2rem" }} variant="h1">
+				Boards
+			</Typography>
+			<Box sx={{ mt: 2, display: "flex", justifyContent: "space-between" }}>
+				<Box sx={{ display: "flex", gap: 1 }}>
+					<Box>
+						<InputLabel sx={{ fontSize: 12, fontWeight: "bold" }} id="sort-by">
+							Sort by
+						</InputLabel>
+						<Select
+							sx={{ width: 200 }}
+							SelectDisplayProps={{ style: { paddingTop: 10, paddingBottom: 8 } }}
+							labelId="sort-by"
+							label="sort by"
+							value={40}
+						>
```

Comment:
update this list to:
const LIST_OPTIONS = [
  {
    label: 'aaa',
    value: 40,
  }
]